tests/Unit/PubSubQueueTests.php

```diff
@@ -16,14 +16,39 @@
 
 class PubSubQueueTests extends TestCase
 {
-    public function teardown()
-    {
-        //
-    }
+    /**
+     * @var string
+     */
+    protected $expectedResult = 'message-id';
+
+    /**
+     * @var Topic
+     */
+    protected $topic;
+
+    /**
+     * @var PubSubClient
+     */
+    protected $client;
+
+    /**
+     * @var Subscription
+     */
+    protected $subscription;
+
+    /**
+     * @var Message
+     */
+    protected $message;
+
+    /**
+     * @var PubSubQueue
+     */
+    protected $queue;
 
     public function setUp()
     {
-        $this->result = 'message-id';
+        $this->expectedResult = 'message-id';
```

Comment:
Property with name `$result` is private and reserved.

https://github.com/sebastianbergmann/phpunit/blob/4c92a15296e58191a4cd74cff3b34fc8e374174a/src/Framework/TestCase.php#L188